config/jobs/kubernetes-sigs/sig-storage-local-static-provisioner/sig-storage-local-static-provisioner.yaml

```diff
@@ -99,6 +99,11 @@ presubmits:
         # this is done so that the e2e test runs faster
         - name: WINDOWS_NODE_OS_DISTRIBUTION
           value: "win2019"
+        # in the e2e tests the flag --working-dir set on CSI Proxy allows
+        # setting the locations where CSI Proxy is supposed to run
+        # the test location is defined in /test/e2e/e2e_test.go
+        - name: CSI_PROXY_FLAGS
+          value: "--working-dir C:\\tmp"
```

Comment:
I know on Linux, /tmp has special behavior regarding permissions that caused some tests to fail. In Windows, is tmp a special type of directory or normal?

Reply:
I believe the directory doesn't exist by default in Windows but it's created for these tests, it's just a regular directory as far as I know